example_iterator.py

```diff
@@ -2,42 +2,43 @@
 
 import settings
 from src.filenames.file_name_generator import OutputFileNameGenerator
-from src.loaders.normalize_scaler_loader import NormalizeScalerLoader
 from src.tif_model_iterator import tif_kernel_iterator
 import glob
 
 if __name__ == "__main__":
     filename = settings.MODEL_PATH
-    tif_file = settings.TIF_FILE
+    tif_file_regex = settings.TIF_FILE_INPUT_REGEX
```

Comment:
You should really update the settings_example.py to reflect this, otherwise the explanation is very unclear when setting up for a new person.